rfc/text/advanced/ap_50_payload_passthru_mode.md

```diff
@@ -0,0 +1,401 @@
+        YIELD.Options.ppt_cipher|string
+        YIELD.Options.ppt_keyid|string
+        ---
+        RESULT.Details.ppt_scheme|string
+        RESULT.Details.ppt_serializer|string
+        RESULT.Details.ppt_cipher|string
+        RESULT.Details.ppt_keyid|string
+        ---
+        ERROR.Details.ppt_scheme|string
+        ERROR.Details.ppt_serializer|string
+        ERROR.Details.ppt_cipher|string
+        ERROR.Details.ppt_keyid|string
+
+{align="left"}
+        PUBLISH.Options.ppt_scheme|string
+        PUBLISH.Options.ppt_serializer|string
+        PUBLISH.Options.ppt_cipher|string
+        PUBLISH.Options.ppt_keyid|string
+        ---
+        EVENT.Details.ppt_scheme|string
+        EVENT.Details.ppt_serializer|string
+        EVENT.Details.ppt_cipher|string
+        EVENT.Details.ppt_keyid|string
+        ---
+        ERROR.Options.ppt_scheme|string
+        ERROR.Options.ppt_serializer|string
+        ERROR.Options.ppt_cipher|string
+        ERROR.Options.ppt_keyid|string
+
+
+**ppt_scheme attribute**
+
+`ppt_scheme` stands for Key Management Schema. It is required string attribute. This attribute can contain name or
+identifier of key management provider which is known to target peer, so it can be used to obtain information
+about encryption keys. *Router* understands that `Payload Passthru Mode` is in use
+by checking the existence and non-empty value of this attribute in `CALL`, `PUBLISH` and `YIELD` messages options.
+
+**ppt_serializer attribute**
+
+`ppt_serializer` attribute is optional. It specifies what serializer was used to build up payload object.
+It can be `mqtt`, `amqp`, `stomp` value just to inform that coming data is from related technologies or ordinary 
+`json`, `msgpack`, `cbor`, `flatbuffers` data serializers.
+
+**ppt_cipher attribute**
+
+`ppt_cipher` attribute is optional. It is required if payload is encrypted. This attribute specifies cryptographic
+algorithm that was used to encrypt payload. It can be `xsalsa20poly1305`, `aes256gcm` for now.
+
+**ppt_keyid attribute**
+
+`ppt_keyid` attribute is optional. This attribute can contain encryption key id that was used to encrypt payload.
+`ppt_keyid` attribute is always a type of string. The value can be a hex-encoded string, uri, DNS name, 
+Ethereum address, UUID identifier - any meaningful value by which target peer can choose private key 
+without guessing. Format of the value may depend on `ppt_scheme` attribute.
+
+**ppt_ predefined schemes**
+
+| Attribute      | Required? | Predefined Scheme (mqtt)  | Predefined Scheme (End-to-End Encryption) | Custom Scheme Example |
+|----------------|-----------|---------------------------|-------------------------------------------|-----------------------|
+| ppt_scheme     | Y         | mqtt                      | wamp.eth                                  | x_my_ppt              |
+| ppt_serializer | N         | json, msgpack, cbor, mqtt | cbor, flatbuffers                         | custom                |
```

Comment:
```suggestion
| ppt_serializer | N         | JSON, Msgpack, CBOR, MQTT | cbor, flatbuffers                         | custom                |
```

Reply:
I think about skipping this, as it shows a value that we use. And under a hidden agreement the values are lowercase

Reply:
Sorry, I thought you were naming those technologies. I see now that they are identifiers, so yes lowercase.